### PR TITLE
libva/2.20.0

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **libva/2.20.0**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/7212397088)

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/7212397088)

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/7212397088)

**Comment by ericLemanissier** [comment]
The advantage this package vs vaapi/system is that it uses conan packages for drm and wayland, instead of system packages.

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/7212397088)

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/7212397088)

File: recipes/libva/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        if self.options.get_safe("with_drm"):
+            self.requires("libdrm/2.4.114")
+        if self.options.get_safe("with_wayland"):
+            self.requires("wayland/1.22.0")
+        if self.options.get_safe("with_glx"):
+            self.requires("opengl/system")
+
+    def validate(self):
+        if self.options.get_safe("with_glx") and not self.options.get_safe("with_x11"):
+            raise ConanInvalidConfiguration(f"{self.ref} needs x11 or glx")
```
Comment (jwillikers) [inline recipes/libva/all/conanfile.py:74]: This message could be clearer, I think.

```suggestion
            raise ConanInvalidConfiguration(f"{self.ref} requires x11 when glx is enabled")
```

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**